queue_job/models/queue_job.py

```diff
@@ -296,7 +296,6 @@ def autovacuum(self):
                 )
                 if jobs:
                     jobs.unlink()
-                    self.env.cr.commit()
```

Comment:
can you explain this change?

Reply:
Hmm. I don’t see this change. It’s not part of my patch.

Reply:
I checked in `git blame`. This comes from 085d5ba8.